pymc/smc/sampling.py

```diff
@@ -420,3 +430,29 @@ def run_chains(chains, progressbar, params, random_seed, kernel_kwargs, cores):
                         )
 
         return tuple(cloudpickle.loads(r.result()) for r in done)
+
+
+def _find_custom_methods(model):
+    custom_methods = {}
+    for rv in model.basic_RVs:
+        rv_type = rv.owner.op
+        cls = rv_type.__class__
```

Comment:
Does this also work for the registry key?
```suggestion
        cls = type(rv_type)
```

If not, what's class exactly?